src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  const isSelected = tab => tab === selectedTab;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({ 'is-active': isSelected(tab) })}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() =>
+                  tab !== selectedTab ? onTabSelected(tab) : null
```

Comment:
The same problem.

Reply:
> The same problem.

It's ok in this case, but I agree that it's not a good idea to compare objects.
Also, you have a function `isSelected` that checks the condition but you don't use it here. And you can just create a variable `isTabSelected` instead of function to avoid repeating the same actions